x-pack/test/functional/apps/maps/index.js

```diff
@@ -17,7 +17,7 @@ export default function ({ loadTestFile, getService }) {
       await esArchiver.load('maps/data');
       await esArchiver.load('maps/kibana');
       await kibanaServer.uiSettings.replace({
-        defaultIndex: 'logstash-*',
+        defaultIndex: 'c698b940-e149-11e8-a35a-370a8516603a',
```

Comment:
A long time ago, we used the index pattern itself as the id, such as `logstash-*`, but ever since 5.6 (I think) we switched to using a GUID for the id of all saved objects.  

The id of an index pattern *could* still be `logstash-*` if it had been created prior to 5.6, or if it were created programatically (like beats setup I think).  

But in most cases, the id is a GUID and when we set the `defaultIndex` we need to use that GUID otherwise it won't work.

Reply:
cc @thomasneirynck